tune_sklearn/_trainable.py

```diff
@@ -51,6 +53,9 @@ def _setup(self, config):
             n_splits = self.cv.get_n_splits(self.X, self.y)
             self.fold_scores = np.zeros(n_splits)
             self.fold_train_scores = np.zeros(n_splits)
+            if not hasattr(self.estimator, "partial_fit"):
+                self.estimator_config["warm_start"] = True
+                self.estimator_config["max_iter"] = 1
```

Comment:
the user can just set this themselves right?

Reply:
Well these are parameters not visible to the user. I thought it'd be better for the user to not worry about whether they're using `partial_fit` or `warm_start`, just if they want to early stop. `max_iter` here is different than the `max_iters` the user sets. `max_iter` is to make sklearn only fit for one epoch, while `max_iters` (which the user can set) is the usual max number of calls to _trainable.

Reply:
OK got it. Can you leave an implementation note as to why you have to do this?